rootfs/api/models/release.py

```diff
@@ -356,7 +356,7 @@ def _delete_release_in_scheduler(self, namespace, version):
         labels = {
             'heritage': 'deis',
             'app': namespace,
-            'version': version
+            'version': "v{}".format(version)
```

Comment:
Look at how `_delete_release_in_scheduler` is used. It seems in some cases it gets a pure version (lacking the `v`) and in other cases it is getting the version from kubernetes labels so it would include the `v`